packages/africa/README.md

```diff
@@ -0,0 +1,73 @@
+# Africa
```

Comment:
Could you add it in the list of maps of the main README and use this icon 🌍 please?